dashboard/models.py

```diff
@@ -18,14 +29,20 @@ class CachedEdxInfoModel(Model):
     """
     user = ForeignKey(User)
     course_run = ForeignKey(CourseRun)
-    data = JSONField(null=True)
-    last_request = DateTimeField()
+    data = JSONField()
 
     class Meta:
         unique_together = (('user', 'course_run'), )
         abstract = True
 
     @classmethod
+    def active_all_qset(cls, user):
```

Comment:
This name and `active_qset` are a bit confusing. Can we rename `active_qset` to `active_all_qset_for_program`? Or something else that distinguishes the two of them